MANIFEST.in

```diff
@@ -0,0 +1 @@
+include harfbuzz/src/*
```

Comment:
a blind include everything under harfbuzz/src may also end up including temporary files generated by the build or similar left-overs found there by accident